opal/corelib/string/encoding.rb

```diff
@@ -210,6 +210,10 @@ def force_encoding(encoding)
 
       if (encoding === self.encoding) { return self; }
 
+      if (typeof self === 'string') {
```

Comment:
Same here but I'm not sure about this fix.
The error is `TypeError: Cannot create property 'encoding' on string ')'`.